src/components/common/Buttons/UserPointButton.tsx

```diff
@@ -27,7 +30,15 @@ const UserPointButton = () => {
             leaveTo='opacity-0 translate-y-1'
           >
             <Popover.Panel className='absolute z-10 mt-1 flex w-screen max-w-max -translate-x-2/3'>
-              <LevelProgressCard />
+              <LevelProgressCard
+                progress={points}
+                level={userLevelData?.level}
+                currentLevel={userLevelData?.currentLevel ?? ''}
+                nextLevel={userLevelData?.nextLevelName ?? ''}
+                pointsNeeded={userLevelData?.pointsNeeded ?? 0}
+                minPoints={userLevelData?.minPoints ?? 0}
```

Comment:
always get these values from the Hook. Default values should be set inside the Hook